src/components/pages/books/books.css

```diff
@@ -185,3 +185,41 @@
 .addBtn:hover {
   cursor: pointer;
 }
+
+@media screen and (max-width: 768px) {
+  .book-section-one {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
```

Comment:
| Zeplin's Design |
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/82db4c04-3e2b-42c8-99b0-75d8772a4a3e) |

| @AleWaweru Design |
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/29931e19-bf17-435a-a3f0-3a3680d8ee65) |

- [ ] It would be nice if you make your Add Book button the same color as in Zeplin's design to make your project follow the design and requirements :+1: